### PR TITLE
fixed issue #5432 using combining carrier_code and method_code

### PR DESCRIPTION
**Pull request by noobCode-69**
**Related issue(s):**
* Fixes [link](https://github.com/scandipwa/scandipwa/issues/5432)

**Problem:**
* Earlier the CheckoutDeliveryOptions was giving error of same key found of multiple components. The reason was 'ups' which is a carrier_code and can be present multiple times.

**In this PR:**
* In this PR i used a combination of carrier_code and method_code to resolve the conflict

### CODE REVIEW

File: packages/scandipwa/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "@scandipwa/service-worker": "0.0.18",
         "@scandipwa/stylelint-config": "0.0.8",
         "@scandipwa/webpack-i18n-runtime": "0.0.36",
-        "@tilework/opus": "0.0.12",
+        "@tilework/opus": "^0.0.12",
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/package.json:21]: Why is this required ?

**Comment by AleksandrsKondratjevs** [comment]
lgtm!
Please take a look at comments.

**Comment by AleksandrsKondratjevs** [comment]
> When first installing the deps , @tilework/opus wasn't install properly , therefore had to manually install

What means not correctly installed ? Nothing has changed in package-lock with you changes

**Comment by noobCode-69** [comment]
I mean when you start the developement server it was giving error that @tilework/opus wasn't installed , therefore had to install it manually!
